src/components/header/header.js

```diff
@@ -0,0 +1,44 @@
+import { useRef } from "react";
+
+import PropTypes from "prop-types";
+
+import * as styledHeader from "./header.module.scss";
+
+const Header = props => {
+  const headerRef = useRef(null);
+  const {
+    sticky = true,
+    stickyMobile = false,
+    maxWidth,
+    backgroundColor,
+    boxShadow,
+    justify,
+    border,
+    ...rest
+  } = props;
+
+  return (
+  <header
+    ref={headerRef}
+  >
+    <div id="header-content" className={styledHeader.container}>
+      {...rest}
```

Comment:
I think this is wrong, what do you try to do here?

Reply:
Header is a useless file, because I deleted the whole folder and nothing happens.

Reply:
Done